[INFO   ] Logger: Record log in D:\GIT\kivy_django_kivy/conf\logs\kivy_22-07-09_51.txt
[INFO   ] deps: Successfully imported "kivy_deps.angle" 0.3.2
[INFO   ] deps: Successfully imported "kivy_deps.glew" 0.3.1
[INFO   ] deps: Successfully imported "kivy_deps.sdl2" 0.4.5
[INFO   ] Kivy: v2.1.0
[INFO   ] Kivy: Installed at "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\__init__.py"
[INFO   ] Python: v3.10.4 (tags/v3.10.4:9d38120, Mar 23 2022, 23:13:41) [MSC v.1929 64 bit (AMD64)]
[INFO   ] Python: Interpreter at "D:\GIT\kivy_django_kivy\kivy_venv\Scripts\python.exe"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 189 symbols loaded
[INFO   ] KivyMD: 0.104.2, git-bc7d1f5, 2021-06-06 (installed at "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivymd\__init__.py")
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.6.0 - Build 30.0.100.9864'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) Iris(R) Xe Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 6
[INFO   ] GL: Shading version <b'4.60 - Build 30.0.100.9864'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <32>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] GL: NPOT texture support is available
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\test2.py", line 64, in <module>
[WARNING] stderr:     TestNavigationDrawer().run()
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\app.py", line 954, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\app.py", line 924, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\test2.py", line 40, in build
[WARNING] stderr:     return Builder.load_file('kv/test2.kv')
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\builder.py", line 305, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\builder.py", line 372, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\parser.py", line 483, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\parser.py", line 593, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\parser.py", line 696, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\parser.py", line 696, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\parser.py", line 696, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   [Previous line repeated 2 more times]
[WARNING] stderr:   File "D:\GIT\kivy_django_kivy\kivy_venv\lib\site-packages\kivy\lang\parser.py", line 706, in parse_level
[WARNING] stderr:     raise ParserException(self, ln,
[WARNING] stderr: kivy.lang.parser.ParserException: Parser: File "D:\GIT\kivy_django_kivy\kv\test2.kv", line 100:
[WARNING] stderr: ...
[WARNING] stderr:      98:
[WARNING] stderr:      99:                    right_action_items:
[WARNING] stderr: >>  100:                        [
[WARNING] stderr:     101:                        ["home", lambda x: app.callback(x), "Home"],
[WARNING] stderr:     102:                        ["coffee", lambda x: app.callback(x), "Coffee order"],
[WARNING] stderr: ...
[WARNING] stderr: Invalid property name
